README.md

```diff
@@ -48,15 +48,27 @@ That things work as the requirements describe. It's okay to ask questions if you
 
 If we like the end result, we will want you to talk us through your code and any decisions you made.
 
-### Running the back end
+## Running the back end
 
-The back end requires Java 8, and can be run from the `backend` directory with the command `./run.sh`
+The back end service can be run from the `backend` directory by following these steps
+1. open a terminal from the `backend` directory
+2. run  `npm ci` to install the dependencies required to run the back end service
+3. after successful packages installation, run `npm run dev`, the starts the back end service.
 
 If it launches correctly you should see the following in the console:
 
-`INFO  [2015-10-29 13:17:10,166] org.eclipse.jetty.server.Server: Started @2126ms`
+```
+  > node-backend@1.0.0 dev
+  > nodemon
+  [nodemon] 2.0.18
+  [nodemon] to restart at any time, enter `rs`
+  [nodemon] watching path(s): src/**/*
+  [nodemon] watching extensions: ts
+  [nodemon] starting `ts-node src/server/server.ts`
+  Server listening on port 6502
+```
 
 This should expose an endpoint on the following URL:
-`http://localhost:6502/application/labourstats`
```

Comment:
Not a big deal, but is there any reason behind removing /labourstats? IMO it represents better a real environment.

Reply:
The route in the ReadMe misses the `labourstats` segment - could you bring it the back? 
The server return things now from `http://localhost:6502/application/labourstats`, the ReadMe still says `http://localhost:6502/application`

For the rest it's good to be merged 👍